src/pages/benchmarks/index.tsx

```diff
@@ -142,6 +142,20 @@ function Benchmarks() {
               */
               }
```

Comment:
^^ Were you meant to delete this commented out code? Lines 125 - 143

Reply:
updated with clarifying comment